examples/plotting/notebook/color_scatterplot.ipynb

```diff
@@ -56,7 +56,7 @@
     "x = np.random.random(size=N) * 100\n",
     "y = np.random.random(size=N) * 100\n",
     "radii = np.random.random(size=N) * 1.5\n",
-    "colors = [\"#%02x%02x%02x\" % (r, g, 150) for r, g in zip(np.floor(50+2*x), np.floor(30+2*y))]"
+    "colors = ["#%02x%02x%02x" % (int(r), int(g), 150) for r, g in zip(50+2*x, 30+2*y)]"
```

Comment:
this change results in the notebook not being able to open with the following error: `Unreadable Notebook: /Users/caged/Dev/bokeh/bokeh/examples/plotting/notebook/color_scatterplot.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n  {\\n   "cell_type": "r...',)`

I'm guessing you need to escape the `"`